### PR TITLE
Update 05-NNSU99.md

### PR DESCRIPTION
**Pull request by NNSU99**

### CODE REVIEW

**Review by NNSU99 — COMMENTED** [review]
Reviewed

File: _slides/05-NNSU99.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
-
-Caption here
-
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+
+my slide 1
```
Comment (NNSU99) [inline _slides/05-NNSU99.md:7]: I add another slide2...nn

Comment (ajo-github) [inline _slides/05-NNSU99.md:7]: Looks fine. 👍

**Review by sammyseltzer — APPROVED** [review]
This looks great.

**Review by ajo-github — COMMENTED** [review]
Reviewed caption change.

**Pull request closed without merging**